tapchannel/aux_sweeper.go

```diff
@@ -2256,26 +2256,20 @@ func (a *AuxSweeper) sweepContracts(inputs []input.Input,
 // sweepExclusionProofGen is a helper function that generates an exclusion
 // proof for the internal key of the change output.
 func sweepExclusionProofGen(sweepInternalKey keychain.KeyDescriptor,
-) tapsend.ExclusionProofGenerator {
+	changeOutputIndex uint32) tapsend.ExclusionProofGenerator {
 
 	return func(target *proof.BaseProofParams,
 		isAnchor tapsend.IsAnchor) error {
 
-		tsProof, err := proof.CreateTapscriptProof(nil)
```

Comment:
Why don't we need to call this any longer? Did it just amount to: 
```go
&proof.TapscriptProof{
					Bip86: true,
				},
```
?

Reply:
Yeah, it's equivalent. But I just liked the more explicit version more, since we explicitly _know_ it's a BIP-86 here.